standard/expressions.md

```diff
@@ -549,7 +549,7 @@ argument_name
 argument_value
     : expression
     | 'ref' variable_reference
-    | 'out' variable_reference
+    | 'out' declaration_expression? identifier
```

Comment:
Action from 2023-04-19:

```suggestion
    | 'out' declaration_expression
    | 'out' variable_reference
```

Reasoning: declaration_expression always has a local variable type, whereas it's fine to use an argument_value of `out x` if `x` is already declared, for example.